samples/drivers/lcd_cyclonev_socdk/src/main.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2022, Intel Corporation
+ * Description:
+ * Example to use LCD Display in Cyclone V SoC FPGA devkit
+ */
+
+#include <zephyr.h>
```

Comment:
zephyr.h and kernel.h below are effectively the same thing, the can both be replaced with <zephyr/kernel.h>